filebeat/docker-compose.yml

```diff
@@ -28,13 +28,13 @@ services:
 
   elasticsearch:
     extends:
-      file: ${ES_BEATS}/testing/environments/${TESTING_ENVIRONMENT}.yml
+      file: ../testing/environments/${TESTING_ENVIRONMENT}.yml
```

Comment:
What is the reason you changed this back?

Reply:
I shouldn't have changed this.

Reply:
bad rebase on my side.